templates/plugin/csharp/pluginClass.cs.tmpl

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using Tizen.Flutter.Embedding;
+using Tizen.System;
```

Comment:
I think the "using System" should be provided basically in this template. Also, it would be better to explicitly add the `Tizen.NET` package to csproj...  I'll fix it.